timescaledb/how-to-guides/continuous-aggregates/migrate.md

```diff
@@ -12,8 +12,32 @@ aggregates created in older versions of TimescaleDB, or created in a new version
 with the option `timescaledb.finalized` set to `false`, use the old format.
 
 To migrate a continuous aggregate from the old format to the new format, you can
-use the provided procedure. Connect to your database and run:
+use this procedure. It automatically copies over your data and and policies.
+
+Connect to your database and run:
 
 ```sql
 CALL cagg_migrate('<CONTINUOUS_AGGREGATE_NAME>');
 ```
+
+## Configure continuous aggregate migration
+
+The migration procedure provides two boolean configuration parameters,
```

Comment:
I think Boolean needs initcaps?

Reply:
Not for the abstract data type, according to Google:

> In most contexts, you're talking about a specific data type in a specific programming language, in which case use code font and the exact spelling and capitalization of the programming keyword. But when referring to the abstract data type, use lowercase. (In the unlikely event that you refer to Boolean mathematics or Boolean logic, use uppercase.)

Reply:
Who knew?! 🤷‍♀️